pkgs/games/deliantra/deliantra-arch.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, deliantra-server }:
+
+stdenv.mkDerivation rec {
+  name = "deliantra-arch-${version}";
+  version = "3.1";
+
+  src = fetchurl {
+    url = "http://dist.schmorp.de/deliantra/${name}.tar.xz";
+    sha256 = "1xzhv48g90hwkzgx9nfjm81ivg6hchkik9ldimi8ijb4j393kvsz";
+  };
+
+  phases = ["unpackPhase" "installPhase"];
+  installPhase = ''
+    mkdir -p "$out"
+    export DELIANTRA_DATADIR="$out"
+    ${deliantra-server}/bin/cfutil --install-arch .
```

Comment:
If it is not very CPU intensive we could just set `hydraPlatforms = []` to build this on the clients itself.

Reply:
Done.